box-browse-sdk/src/main/java/com/box/androidsdk/browse/adapters/BoxItemAdapter.java

```diff
@@ -591,32 +606,37 @@ protected void onBindBoxItemViewHolder(BoxItemViewHolder holder, BoxItem itemToB
          * @return String Short human readable String e.g. 2.5 MB
          */
         private String localFileSizeToDisplay(final double numSize) {
+            if (numSize == 0){
+                return SIZE_0_BYTES;
+            }
             final int constKB = 1024;
             final int constMB = constKB * constKB;
             final int constGB = constMB * constKB;
             final double floatKB = 1024.0f;
             final double floatMB = floatKB * floatKB;
             final double floatGB = floatMB * floatKB;
-            final String BYTES = "B";
-            String textSize = "0 bytes";
+            String textSize = SIZE_0_BYTES;
             String strSize = Double.toString(numSize);
             double size;
 
             if (numSize < constKB) {
-                textSize = strSize + " " + BYTES;
+                textSize = strSize + " " + SIZE_BYTES;
             } else if ((numSize >= constKB) && (numSize < constMB)) {
                 size = numSize / floatKB;
-                textSize = String.format(Locale.getDefault(), "%4.1f KB", size);
+                textSize = String.format(Locale.getDefault(), "%4.1f ", size) + SIZE_KILOBYTES;
```

Comment:
I don't think this is done yet.... to be completely localizable, we would need to include this whole formatting into the string. Then the localization team can work out other languages format. The way we are doing here right now does not cover for instance RTL languages.